plugins/dapper/surfactantplugin_dapper.py

```diff
@@ -0,0 +1,183 @@
+import logging
+from dataclasses import asdict, dataclass
+from pathlib import Path
+from typing import Any, Dict, List, Optional
+
+import surfactant.plugin
+from surfactant.sbomtypes import SBOM, Software
+
+try:
+    from dapper_python.databases.linuxDB import LinuxDB
+    from dapper_python.dataset_loader import DatasetCatalog
+    from dapper_python.normalize import NormalizedFileName, normalize_file_name
+
+    DAPPER_AVAILABLE = True
+except ImportError:
+    DAPPER_AVAILABLE = False
+    logging.warning("dapper-python not installed. Dapper plugin will be disabled.")
```

Comment:
If the plugin was pip installed, this dependency should always be available because it is listed as a dependency in pyproject.toml and therefore pip will install it. The only time I can think of it not being present is if someone did some strange manual installation of the plugin by copying it into their site-packages folder and side stepped pip entirely.